capa/main.py

```diff
@@ -598,6 +615,11 @@ def get_file_extractors(sample: str, format_: str) -> List[FeatureExtractor]:
     elif format_ == capa.features.extractors.common.FORMAT_ELF:
         file_extractors.append(capa.features.extractors.elffile.ElfFeatureExtractor(sample))
 
+    elif format_ == FORMAT_CAPE:
+        with open(sample, "rb") as f:
+            report = json.load(f)
+        file_extractors.append(capa.features.extractors.cape.extractor.CapeExtractor.from_report(report))
```

Comment:
should we add a `from_file` helper function, to do this based on a provided `pathlib.Path`?

Reply:
the original `from_file` method was with regards to submitting an executable sample for analysis rather than using the ready report. personally, i think a helper function for getting a report `dict` from a json filename does make sense to me since that code snippet seems to repeat a lot.